cmd/commonflags.go

```diff
@@ -26,6 +27,7 @@ func addBasicFlags(cmdset *pflag.FlagSet) {
 func addCommonFlags(cmdset *pflag.FlagSet) {
 	addBasicFlags(cmdset)
 	cmdset.StringVar(&mode, "mode", "count", "coverage mode: set, count, atomic")
+	cmdset.StringVar(&agentPort, "agentport", "", "specify fixed port for registered service")
```

Comment:
suggest to enhance usage, like:
`the port to listen when the registered service starting, if not provided, it will listen a random one `

Reply:
update:
specify fixed port for registered service communicate with goc server. if not provided, using a random port